tests/memray/Makefile

```diff
@@ -0,0 +1,15 @@
+profiles:
+	mkdir -p .profiles
```

Comment:
I'm surprised these are still showing up in the diff, you may have to merge from main?

Reply:
completed